README.md

```diff
@@ -8,16 +8,165 @@ Local.
 For more information, see [Getting started with Cloudant NoSQL DB](https://www.ng.bluemix.net/docs/services/Cloudant/index.html)
 
 The `loopback-connector-cloudant` module is the Cloudant connector for the Loopback framework.
+<!-- TOC -->
+
+- [loopback-connector-cloudant](#loopback-connector-cloudant)
```

Comment:
@ssh24 oh ok I see, true, I have auto TOC generator on when saving my file, not sure how to tell the generator skip it lol. I will disable the generator and delete it in the final commit.